Jenkinsfile_CNP

```diff
@@ -29,11 +29,12 @@ static LinkedHashMap<String, Object> secret(String secretName, String envVar) {
 // THESE WILL BE OVERWRITTEN IN ASE DEPLOYMENTS WITH TERRAFORM OUTPUTS.TF
 // THESE CAN BE PICKED UP IN INTEGRATION-TEST CODE DIRECTLY
 env.IDAM_URL = 'https://idam-api.aat.platform.hmcts.net'
-env.CLAIM_STORE_URL = 'http://cmc-claim-store-aat.service.core-compute-aat.internal'
+env.CLAIM_STORE_URL = 'http://cmc-claim-store-aat-staging.service.core-compute-aat.internal'
```

Comment:
not sure of this, this was originally for AKS based functional tests, this change is not helping anything

Reply:
That is correct. I will try to remove it after a first successful build to be honest. Thanks